crates/graphql-federated-graph/src/federated_graph.rs

```diff
@@ -99,6 +127,20 @@ pub struct Subgraph {
     pub url: Option<StringId>,
 }
 
+#[derive(Clone)]
+pub struct Extension {
+    /// Name of the extension within the federated graph. It does NOT necessarily matches the extension's name
+    /// in its manifest, see the `id` field for this.
+    pub enum_value_name: StringId,
+    pub url: StringId,
+
+    // -- loaded from the extension manifest --
+    #[cfg(feature = "extension")]
+    pub id: extension::Id,
+    #[cfg(feature = "extension")]
+    pub manifest: extension::Manifest,
```

Comment:
Should this also be an id?

Reply:
Shouldn't be needed, extensions definitions in the `extesion__Link` enum should be unique